### PR TITLE
Top Followed section, assets, user page improvements, messages UI updates

### PR DESCRIPTION
**Pull request by micahlt**
In this PR: 

- Optimized images
- Added a Top Followed Users section to the bottom of Explore
- Switched to approximate numbers for loves, favorites, remixes, and followers
- Added assets for Google Play
- Added a clear divider between read and unread messages
- Multiple UI bugfixes
- Added projects to the user page
- Support for Scratch Team notifications (mutes, bans, etc) on Messages page

### CODE REVIEW

File: src/views/ExplorePage.vue [hunk 1]
```diff
@@ -287,6 +304,25 @@ export default {
           }
         });
       }
+      Http.request({
+        method: "GET",
+        url: "https://scratchstats.com/api2/scratchdb/topusers/followers?country=global&page=0",
+        headers: {
+          Referer: "https://scratchstats.com/",
+          "User-Agent":
```
Comment (locness3) [inline src/views/ExplorePage.vue:312]: Why is the UA modified?

**Comment by locness3** [comment]
> * Support for Scratch Team notifications (mutes, bans, etc) on Messages page

How'd you test this feature? :)

**Comment by locness3** [comment]
Any apk too?

File: src/views/ExplorePage.vue (hunk 1 above)
Comment (micahlt) [reply src/views/ExplorePage.vue:312]: Honestly not sure - probably was trying to fix some network bug.

**Comment by micahlt** [comment]
> > * Support for Scratch Team notifications (mutes, bans, etc) on Messages page
> 
> How'd you test this feature? :)

**Whoops**.

> Any apk too?

I'm about to release this, but I have to compile the APK.  I'll drop an @mention when it's released.

**Comment by micahlt** [comment]
@locness3 Release APK is out: https://github.com/Scratch-Client-4/itchy-ionic/releases/tag/0.9.5#assets

File: src/views/ExplorePage.vue (hunk 1 above)
Comment (locness3) [reply src/views/ExplorePage.vue:312]: Well, please remove this.

**Comment by locness3** [comment]
> Added a Top Followed Users section to the bottom of Explore

I'm not exactly sure what's the point of this feature, mostly because this data isn't changing really much regularly - you come to the Explore page to see new stuff, not to check that griffpatch is still on the 1st place IMO. Would appreciate if you asked before adding this - just like you should be doing with most significant changes, or at least you should not merge PRs before approval.

> Added a clear divider between read and unread messages

I believe read messages could be hidden behind either a toggle or a link to a dedicatedr page - have to play around with this. This would require a mark as read button too.

> Added projects to the user page
> Support for Scratch Team notifications (mutes, bans, etc) on Messages page

LGTM.

> Optimized images
> Multiple UI bugfixes

Could you be more precise about those?

**Comment by micahlt** [comment]
> I'm not exactly sure what's the point of this feature

Is there harm in including more data (I mean, performance-wise yes, but we're not there yet)?  I incorporated this feature because I like to check up on some of the most popular Scratchers and their projects, and it's handy to have a list there without having to go to ScratchStats.  

> I believe read messages could be hidden behind either a toggle or a link to a dedicated page

Why?  What's your reasoning here?  We have to get all (both read and unread) messages simultaneously, so what's the benefit of hiding messages behind another interaction?

> Could you be more precise about those?

Merged an ImgBot PR that'll optimize some assets (mainly the Google Play assets uploaded in this PR).  UI bugfixes mainly refers to me fixing scrolling in project/user modals.  

> Would appreciate if you asked before adding this

To be fair this is my project, and very few people are currently showing interest in contributing.  I thought you'd abandoned the project, which is why I didn't wait for feedback before merging.

**Comment by locness3** [comment]
> Why? What's your reasoning here? We have to get all (both read and unread) messages simultaneously, so what's the benefit of hiding messages behind another interaction?

When I go to the app I either want to see my read or unread messages, rarely both at the same time. But perhaps that's just me.

**Pull request merged**